### PR TITLE
Map

### PR DESCRIPTION
**Pull request by khaliqzohaib92**
- Updated development database by importing protected data from the staging environment through pgAdmin4.
- Updated the user roles for updated development database. 
- Restricted the population percentage resizing to zoom level 9, to avoid visual overlap of multiple dots in the zoomed-out view. 
- Corrected the zero count issue by updating the active record query in the user model for total counts.
- Added zero count check for all user types to switch statements within isValidCount function. 
- Validated zoom functionality, marker sizing functionality, and database updating. 

**Suggestions to Improve circle marker visualization:** 
- Currently, the city name allows duplicate cities because of case insensitivity. This is causing an overlap of dots within certain cities and unnecessarily duplicated data on the map. A possible solution is to restrict cities to a drop-down menu to make city names input uniform. 

**Current Limitation**
- In the zoomed-out view in order to avoid overlap, we have to keep the marker size small. The tradeoff is the dots are difficult to click on.  If we were to increase the size of the marker to make them easier to click they will overlap causing a jumble of information and messy visualization. 

image  
- With the react-leaflet map library we can not keep the different sized markers on the zoomed-in view from overlapping. The tradeoff would be removing the population sizing, making all dots uniform.

image

### CODE REVIEW

File: app/javascript/components/reusable/withUserForm.js [hunk 1]
```diff
@@ -58,15 +58,20 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
       this.resetForm = this.resetForm.bind(this);
       this.changeCountryHandler = this.changeCountryHandler.bind(this);
       this.changeStateHandler = this.changeStateHandler.bind(this);
+<<<<<<< HEAD
```
Comment (kscali) [inline app/javascript/components/reusable/withUserForm.js:61]: Can you remove this merge issue we have going on?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,13089 @@
+{
```
Comment (kscali) [inline package-lock.json:1]: Can you delete this package-lock.json? We use yarn for our package installs. Having both a yarn.lock and a package-lock.json will break the app when deployed to Passsenger.

**Review by kscali — CHANGES_REQUESTED** [review]

File: app/javascript/components/reusable/withUserForm.js (hunk 1 above)
Comment (khaliqzohaib92) [reply app/javascript/components/reusable/withUserForm.js:61]: Sure, will update when done.

File: package-lock.json (hunk 2 above)
Comment (khaliqzohaib92) [reply package-lock.json:1]: Sure, sorry it was created while working with seeding the PostgreSQL by switching between PowerShell and WSL.

**Pull request merged**